app/models/salt_handler/orchestration_trigger.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# This class is responsible to handle the salt events that are orchestration triggers.
+# Only the bootstrap and update orchestrations will be handled by this class.
+class SaltHandler::OrchestrationTrigger
```

Comment:
can we add a base class (e.g. `SaltHandler::Orchestration::Base` or just `SaltHandler::Orchestration`) so that `SaltHandler::Orchestration::Trigger` and `SaltHandler::Orchestration::Result` can inherit from which contains the class methods:

* self.can_handle_event?
* self.salt_fun_args_match

and the constants? this would be easy to do i think and get rid of the duplication